### PR TITLE
slice simple object

### PR DESCRIPTION
**Pull request by Guseyn**
Add slicee simple from the ticket #73

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#79](https://github.com/artipie/http/issues/79) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/7f2822d6-d253-4dbe-8448-932d313ed633); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/http/slice/SliceSimple.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+package com.artipie.http.slice;
+
+import com.artipie.http.Response;
+import com.artipie.http.Slice;
+import java.nio.ByteBuffer;
+import java.util.Map;
+import org.reactivestreams.Publisher;
+
+/**
+ * SliceSimple.
```
Comment (Vatavuk) [inline src/main/java/com/artipie/http/slice/SliceSimple.java]: @Guseyn add more appropriate description. Describe what is this class used for.

File: src/main/java/com/artipie/http/slice/SliceSimple.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ *
+ * @since 0.1
```
Comment (g4s8) [inline src/main/java/com/artipie/http/slice/SliceSimple.java]: @Guseyn please use next minor version

File: src/main/java/com/artipie/http/slice/SliceSimple.java (hunk 1 above)
Comment (Guseyn) [reply src/main/java/com/artipie/http/slice/SliceSimple.java]: @Vatavuk done.

File: src/main/java/com/artipie/http/slice/SliceSimple.java (hunk 2 above)
Comment (Guseyn) [reply src/main/java/com/artipie/http/slice/SliceSimple.java]: @g4s8 done.

**Comment by Guseyn** [comment]
@Vatavuk @g4s8 plz see the updates above.

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @vatavuk[/z](https://www.0crat.com/u/vatavuk)[/z](https://www.0crat.com/u/vatavuk) didn't make CR comments

**Comment by 0crat** [comment]
Job was finished in 17 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Pull request merged**